app/pages/subscription-hub.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAnchorWallet } from '@solana/wallet-adapter-react';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useUser } from '../hooks/userUser';
+
+const SubscriptionHub: NextPage = () => {
+  const router = useRouter();
+  const connectedWallet = useAnchorWallet();
+  const { user } = useUser();
+
+  // Reroute to `/` if wallet not connected.
+  !connectedWallet && router.push('/');
+
+  // Reroute to `/creator-hub` if the user is a creator.
+  user.creatorAccount && router.push('/creator-hub');
+
+  // Reroute to `/subscribe` if user hasn't subscribed to any content.
+  !user.subscriptions.length && router.push('/subscribe');
```

Comment:
in this case we can reroute them to the landing page. we won't have a `/subscribe` route, they'll have to go directly to a creator landing page to subscribe to a creator

Reply:
I see. I thought creator landing page was specific to particular creator. I meant that only by `/subscribe`.

Reply:
Correct, each creator gets their own creator landing page

Reply:
By `/subscribe` I meant a page which would show different creators. IG we can call that `/creators` ? And reroute the user to that page ?

Reply:
yea totally 👍  that sounds good